app/services/order_tax_adjustments_fetcher.rb

```diff
@@ -36,6 +36,11 @@ def line_item_adjustments
       .and(table[:adjustable_type].eq('Spree::LineItem'))
   end
 
+  def shipment_adjustments
+    table[:order_id].eq(order.id)
+      .and(table[:adjustable_type].eq('Spree::Shipment'))
+  end
```

Comment:
:ok_hand: :ok_hand: it's a pity, though, because it seems that it's not until Rails 5 that AR's `#or` can take an AR relation as an argument. This could then become a relation instead of an Arel query. Reference: https://apidock.com/rails/v5.0.0.1/ActiveRecord/QueryMethods/or

Reply:
Yeah, there's lots of places where we can benefit from that as soon as we get to Rails 5, I keep running in to it.